BankingAppControllers/Controllers/AccountController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using BankingAppApiModels.Models.Account;
+using BankingAppBusiness.AccountRepo;
+using Microsoft.AspNetCore.Mvc;
+namespace BankingAppControllers.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class AccountController : ControllerBase
+    {
+        private readonly IAccountRepository _account;
+        public AccountController(IAccountRepository account)
+        {
+            _account = account;
+        }
+        [HttpPost("/addAccount")]
```

Comment:
You can remove "/addAccount" the difference will be between http verbs. Please remove unnecessary relativeUrls